.github/workflows/e2e.yml

```diff
@@ -8,10 +8,10 @@ jobs:
         uses: actions/checkout@v1
       # Install NPM dependencies, cache them correctly
       # and run all Cypress tests
+      - name: Install dependencies
+        run: yarn
       - name: Cypress run
-        uses: cypress-io/github-action@v1
-        with: 
-          browser: chrome
+        run: yarn run cypress run
```

Comment:
Shouldn't it be `yarn run cypress:run`? Alternatively `yarn cypress:run`

Reply:
Maybe it should, but it works well now so I wouldn't touch it :D